src/main/java/org/apache/accumulo/proxy/ProxyServer.java

```diff
@@ -188,10 +183,7 @@ public void onRemoval(RemovalNotification<UUID,ConditionalWriter> notification)
 
   public ProxyServer(Properties props) {
 
-    @SuppressWarnings("deprecation")
-    org.apache.accumulo.core.client.Instance i = new org.apache.accumulo.core.client.ZooKeeperInstance(
-        ClientConfConverter.toClientConf(props));
-    instance = i;
+    client = Accumulo.newClient().from(props).build();
 
     try {
       String tokenProp = props.getProperty("tokenClass", PasswordToken.class.getName());
```

Comment:
I don't think you need this check anymore. `Accumulo.newClient()` should be able to set up the `client` using whatever token type it supports. That restriction was in place before because the preferred language client could only support passwords... but now that we're not logging in using the preferred language client, and the proxy is setting up the client on its own, we don't need to restrict it.